src/core/api.proto

```diff
@@ -94,6 +135,14 @@ message InferRequestHeader
     //@@       for tensors with a non-fixed-size datatype (like STRING).
     //@@
     uint64 batch_byte_size = 3;
+
+    //@@    .. cpp:var:: InferSharedMemory shm_io
+    //@@
+    //@@       It is the location in shared memory to read the input tensors.
```

Comment:
"... location in shared memory that contains the tensor data for this input"

shm_io -> shared_memory